### ISSUE TITLE
1.13.2 breaks sites, fatal error

### ISSUE THREAD

**Issue by kebbet** [johnbillion/wp-crontrol#84]
Admin notice says:
`Warning: require_once(/home/user/sites/**/public_html/wp-content/plugins/wp-crontrol/vendor/autoload.php): failed to open stream: No such file or directory in /home/user/sites/**/public_html/wp-content/plugins/wp-crontrol/wp-crontrol.php on line 45`

` Fatal error: require_once(): Failed opening required '/home/user/sites/**/public_html/wp-content/plugins/wp-crontrol/vendor/autoload.php' (include_path='.:/opt/alt/php74/usr/share/pear') in /home/user/sites/**/public_html/wp-content/plugins/wp-crontrol/wp-crontrol.php on line 45`

**Comment by terrapop** [comment]
Same here.

**Comment by TheEMINeM** [comment]
same here! I hope vendors will fix this asap

**Comment by TheEMINeM** [comment]
Rolled back the old version until coders releases a fix. :)

**Comment by mattbedford** [comment]
Same here..... sigh.

**Comment by fischerpu** [comment]
Same here. Rolled back to previous version resolves the issue.

**Comment by ezkayaker** [comment]
Yes.  I have renamed the plugin folder wp-crontrol to wp-crontrol.bak. The plugin just took my site down on update to new version.

**Comment by Regnalf** [comment]
A quick temporary fix for all who need a solution right now
https://wordpress.org/support/topic/a-fixed-zip-for-fatal-error-missed-vendor-folder/

**Comment by mahmoudgad94** [comment]
Just fire inside folder plugin (or fire it local and upload to server)
```
composer install --no-dev
```

**Comment by rleeson** [comment]
> Just fire inside the folder plugin (or fire it local and upload it to the server)
> 
> ```
> composer install --no-dev
> ```

This or `composer dump-autoload` on the server if available or locally then upload are the best way to handle temporarily patching the issue until the team resolves the CI issue.

**Comment by rleeson** [comment]
It looks like the tagged release on this repository is correct and you can also download and install that directly. The ZIP downloaded from the WP plugins directory is missing the `vendor` directory for some reason. Is there a separate build workflow for that which doesn't use the Github Workflow?

**Comment by dc0063** [comment]
The error is from the following requirement at the beginning:    require_once __DIR__ . '/vendor/autoload.php';
as this references the plug-in folder of wp-crontrol as might be expected.  However, the path is incorrect as the file is in the folder - wp-optimize, not wp-crontrol - ANOTHER PLUG-IN.
Ooops.  
Disabled, waiting for the update

**Comment by MichaIng** [comment]
Here is a fix to run from console until a new version has been released:
```sh
cd /tmp
curl -LO https://github.com/johnbillion/wp-crontrol/archive/refs/tags/1.13.2.tar.gz
tar xf 1.13.2.tar.gz
mv wp-crontrol-1.13.2/vendor /var/www/wordpress/wp-content/plugins/wp-crontrol/
rm -R 1.13.2.tar.gz wp-crontrol-1.13.2
```
Adjust the path to your Wordpress accordingly, of course.

**Comment by dakman** [comment]
yep effected us too... sucks cause it isnt recoverable.. eg takes the entire instance down  and wp-auto updates probably wouldnt run since its a fatal error

**Comment by johnbillion** [comment]
Thanks for the reports everyone. This is an unexpected bug in the GitHub action that I use to package the plugin for the wordpress.org plugin directory. I'm looking into options to fix it.

**Comment by mjgardner** [comment]
> A quick temporary fix for all who need a solution right now https://wordpress.org/support/topic/a-fixed-zip-for-fatal-error-missed-vendor-folder/

Page not found error on that link

**Comment by johnbillion** [comment]
I've released version 1.14 which reverts the changes made in 1.13 (_"Unlucky for some"_).

I'll investigate the issue with the wordpress.org deployment, I can't immediately see why the vendor directory is missing because it's been accounted for in the build process.
